dev-guide/ci/events-files.md

```diff
@@ -0,0 +1,64 @@
+# Openshift CI produced files related to events
+
+To aid in debugging, the Openshift CI (prow) produces various files related to events.  The files
+reside in the gather-must-gather and gather-extra subdirectories in the artifacts storage area.
+You will see the "Artifacts" link in the upper right of a prow job (for example, see
+[this prow job](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664).
+
+This document provides details on where the files are located and what they contain.
+
+## Events files
+
+* event-filter.html
+  * included in the output of a `oc adm must-gather`
+  * contains the events from all namespaces plus events that exist in the kube-apiserver at the time `oc adm must-gather` was run
```

Comment:
not all events, just the events in namespaces gathered by must-gather.  so namespaces not listed as relatedResources for clusteroperators will be missing.